mklive.sh.in

```diff
@@ -374,7 +374,7 @@ if [ -n "$LINUX_VERSION" ]; then
     _linux_series="$LINUX_VERSION"
     PACKAGE_LIST="$PACKAGE_LIST $LINUX_VERSION"
 else # Otherwise find latest stable version from linux meta-package
-    _linux_series=$(XBPS_ARCH=$BASE_ARCH $XBPS_QUERY_CMD -r "$ROOTFS" ${XBPS_REPOSITORY:=-R} -x linux|head -1)
+    _linux_series=$(XBPS_ARCH=$BASE_ARCH $XBPS_QUERY_CMD -r "$ROOTFS" ${XBPS_REPOSITORY:=-R} -x linux|tail -1)
```

Comment:
`head` and `tail` sound a bit too fragile, do you think `| grep 'linux[0-9]'` could be better?

Reply:
yes. I think it's better